readthedocs/projects/tasks/builds.py

```diff
@@ -796,12 +796,22 @@ def store_build_artifacts(self):
         types_to_delete = []
 
         for artifact_type in ARTIFACT_TYPES:
-            if os.path.exists(
-                self.data.project.artifact_path(
-                    version=self.data.version.slug,
-                    type_=artifact_type,
+            artifact_directory = self.data.project.artifact_path(
+                version=self.data.version.slug,
+                type_=artifact_type,
+            )
+            if os.path.isdir(artifact_directory):
```

Comment:
Don't we want this to be a directory? Won't we basically be checking if `_readthedocs/pdf` is a directory, which we want? I think we want to check it's _not_ a directory, and if it has multiple files in it, if it is?

Reply:
Yes. You already found that I've already solve this problem in the following PR.